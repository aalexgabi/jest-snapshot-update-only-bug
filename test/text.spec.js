
describe('group', () => {
    describe('subgroup', () => {
        test('should keep this one', () => {
            expect('hello').toMatchSnapshot('snap');
        })

        test('should update this one', () => {
            expect('bonjour').toMatchSnapshot('snap');
        })
    })
})
