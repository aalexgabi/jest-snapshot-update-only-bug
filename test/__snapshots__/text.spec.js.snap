// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`group subgroup should keep this one: snap 1`] = `"hello"`;

exports[`group subgroup should update this one: snap 1`] = `"bonjour"`;
